distro/wildfly/subsystem/src/main/java/org/camunda/bpm/container/impl/jboss/extension/handler/JobExecutorAdd.java

```diff
@@ -45,56 +47,54 @@
  */
 public class JobExecutorAdd extends AbstractAddStepHandler {
 
-    public static final String THREAD_POOL_GRP_NAME = "Camunda BPM ";
-
-    public static final JobExecutorAdd INSTANCE = new JobExecutorAdd();
+  public static final String THREAD_POOL_GRP_NAME = "Camunda BPM ";
 
-    private JobExecutorAdd() {
-        super(SubsystemAttributeDefinitons.JOB_EXECUTOR_ATTRIBUTES);
-    }
+  public static final JobExecutorAdd INSTANCE = new JobExecutorAdd();
 
-    @Override
-    protected void performRuntime(final OperationContext context, final ModelNode operation, final ModelNode model)
-            throws OperationFailedException {
+  @Override
+  protected void performRuntime(final OperationContext context, final ModelNode operation, final ModelNode model)
+      throws OperationFailedException {
 
-        String jobExecutorThreadPoolName =
-                SubsystemAttributeDefinitons.THREAD_POOL_NAME.resolveModelAttribute(context, model).asString();
-        ServiceName jobExecutorThreadPoolServiceName = ServiceNames.forManagedThreadPool(jobExecutorThreadPoolName);
+    String jobExecutorThreadPoolName = SubsystemAttributeDefinitons.THREAD_POOL_NAME
```

Comment:
💬 I wonder why `THREAD_POOL_NAME` is deprecated. This is our code. Maybe we should remove the deprecation if we still need to use the constant?

Reply:
I checked the history, it seems the annotation was added by mistake for the thread pool name: https://github.com/camunda/camunda-bpm-platform/commit/fa228e273fc0e43be2142d3883345ff73770d2de
I will remove the annotation.